### PR TITLE
device pages on mobile

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers

### CODE REVIEW

**Comment by MarcoPolo** [comment]
I'm going to run this, but screenshots would be cool so we have an image attached to this diff for the archive

File: shared/devices/device-page/dumb.native.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  name: 'Home Computer',
+  isCurrent: false,
+  isRevoked: false,
+  timeline: null,
+  banner: null,
+  onRevoke: () => {}
+}
+
+export default {
+  'Device Page': {
```
Comment (MarcoPolo) [inline shared/devices/device-page/dumb.native.js]: 🐑 Would be nice to use the type https://github.com/keybase/client/blob/master/shared/constants/types/more.js#L24

Example: https://github.com/keybase/client/blob/master/shared/common-adapters/dumb.native.js#L8

**Comment by chrisnojima** [comment]
yeah i usually do but i forgot

File: shared/local-debug.native.js [hunk 2]
```diff
@@ -34,8 +34,8 @@ if (__DEV__ && true) {
   config.printOutstandingRPCs = true
   config.reactPerf = false
   config.overrideLoggedInTab = Tabs.moreTab
-  config.dumbFilter = ''
-  config.dumbIndex = 0
+  config.dumbFilter = 'device page'
```
Comment (MarcoPolo) [inline shared/local-debug.native.js]: Don't check this in

**Comment by MarcoPolo** [comment]
Badge is a little off on android:

screen

We can make a separate ticket and come back to it

**Comment by chrisnojima** [comment]
yeah thats a function of the text. we have a ticket to fixup styling issues like this already so lets hit it then

**Comment by chrisnojima** [comment]
screen shot 2016-04-26 at 4 12 48 pm
screen shot 2016-04-26 at 4 13 16 pm
screen shot 2016-04-26 at 4 13 33 pm
screen shot 2016-04-26 at 4 13 49 pm

**Comment by chrisnojima** [comment]
@MarcoPolo fixed

**Comment by MarcoPolo** [comment]
👍

**Pull request merged**